### PR TITLE
Fixes #37231 - Append domain name setting follow-up

### PR DESCRIPTION
**Pull request by Dyrkon**
Follow-up fixes.

### CODE REVIEW

File: app/models/host/base.rb [hunk 1]
```diff
@@ -372,7 +372,11 @@ def to_label
       if Setting[:display_fqdn_for_hosts]
```
Comment (stejskalleos) [inline app/models/host/base.rb]:
```suggestion
      return name if Setting[:display_fqdn_for_hosts]
```

File: app/models/host/base.rb [hunk 2]
```diff
@@ -372,7 +372,11 @@ def to_label
...
         name
       else
-        name.split('.')[0]
+        if domain.nil?
```
Comment (stejskalleos) [inline app/models/host/base.rb]:
```suggestion
return name unless domain
```

**Review by stejskalleos — CHANGES_REQUESTED** [review]

**Comment by stejskalleos** [comment]
I see it's still in WIP, any blockers here?

**Comment by Dyrkon** [comment]
> I see it's still in WIP, any blockers here?

Yes, there is some internal dispute regarding the way we are going to resolve this.

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

Works as expected

**Comment by stejskalleos** [comment]
Thanks @Dyrkon

**Pull request merged**